cluster-scope/base/core/namespaces/koku-metrics-operator/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: koku-metrics-operator
+resources:
+- namespace.yaml
```

Comment:
needs a new line at the end of the file. 

also, a few other files in this PR need newline.